spec/acceptance/redis_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+skip = if (fact('os.family') == 'RedHat') && (fact('os.name') != 'Fedora') && !ENV.fetch('ICINGA_REPO_USER', nil).nil? && !ENV.fetch('ICINGA_REPO_PASSWORD', nil).nil?
```

Comment:
Should it test for the `icinga_repo_user` and `icinga_repo_password` facts instead of env vars?

Reply:
Sure, that was the initial idea.